x11-misc/xkblayout-state/xkblayout-state-20171214.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit toolchain-funcs
+
+# Commit Date: 27 Nov 2017
+COMMIT="d023bd7f58a54da08c3aad3c234a629b8a010892"
+
+DESCRIPTION="A small program to get/set the current XKB layout"
+HOMEPAGE="https://github.com/nonpop/xkblayout-state"
+SRC_URI="${HOMEPAGE}/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
```

Comment:
Don't use HOMEPAGE here. It is multi-valued and you don't want the ebuild to fail randomly when someone adds second homepage.

Reply:
Done.